### DISCUSSION TITLE
添加多国语言的教程为什么没有了？

### DISCUSSION THREAD

**Discussion by fw6x** [hiroi-sora/Umi-OCR#536] [Q&A]
添加多国语言的教程为什么没有了？

**Comment by hiroi-sora** [comment]
因为常用的多国语言已经默认附带在软件包里了。

没有附带的语言，要不就是效果差没有实用价值，要不就是Paddle还没有发布对应模型库。

如果需要识别小众语言，建议使用 TesseractOCR插件，它支持众多小语种。相关教程请见：

https://github.com/qwedc001/tesseractOCR_umi_plugin
